database/src/main/java/info/metadude/android/eventfahrplan/database/repositories/RealHighlightsDatabaseRepository.kt

```diff
@@ -77,6 +78,23 @@ class RealHighlightsDatabaseRepository(
         }
     }
 
+    /**
+     * Resets the value of the [HIGHLIGHT] column to [`false`][HIGHLIGHT_STATE_OFF] for the row
+     * with the given [sessionId].
+     */
+    override fun delete(sessionId: String): Int {
+        with(sqLiteOpenHelper) {
+            return writableDatabase.updateRow(
```

Comment:
Why update the row and not delete it?

Reply:
Consistency with `deleteAll()` for now. Nothing I can explain right now. I leave the investigation and possible cleanup for a follow-up pull request. I linked the related commit 8bee70b0aa1974e56610cde819f7df04ac8f92e7 in the commit message.